pkg/storegateway/partitioner.go

```diff
@@ -42,6 +43,10 @@ func newGapBasedPartitioner(maxGapBytes uint64, reg prometheus.Registerer) *gapB
 			Name: "cortex_bucket_store_partitioner_expanded_ranges_total",
 			Help: "Total number of byte ranges returned by the partitioner after they've been combined together to reduce the number of bucket API calls.",
 		}),
+		extendedRangesRanges: promauto.With(reg).NewCounter(prometheus.CounterOpts{
+			Name: "cortex_bucket_store_partitioner_extended_ranges_total",
+			Help: "Total number of byte ranges that were not overlapping but were joined because they were closer than maxGapBytes.",
```

Comment:
```suggestion
			Help: "Total number of byte ranges that were not overlapping but were joined because they were closer than the configured maximum gap.",
```